### ISSUE TITLE
Reinforce Accessibility!

### ISSUE THREAD

**Issue by seth-way** [CarissaHluchan/whats-cookin#29]

**Comment by seth-way** [comment]
- You must be able to tab through your app and use it without a mouse
- Your app must still be usable when tested with a colorblind extension
- You must score as close to 100% as possible with the “Lighthouse” and “WAVE” accessibility audits. Make sure that these audits pass on all pages and views. Be prepared to explain any accessibility audits your application is failing.
- Your HTML must be written semantically and should use ARIA tags (ONLY if needed / appropriate)
